newIDE/app/src/EventsSheet/EventsTree/helpers.js

```diff
@@ -75,11 +62,6 @@ export const moveNodeAbove = ({ targetNode, node }: MoveFunctionArguments) => {
     initialEventsList: node.eventsList,
     toIndex,
   });
-  const previousRowIndex = getRowIndexOfNode(node);
```

Comment:
We used the return value for history. It was useful at the time and easier to compute, but since we allow changing depth, we have now no real idea of where the node ends up being.

For example, here we can't easily determine where the node goes knowing only targetNode (which is just above the dragged node) and the dragged node.
https://user-images.githubusercontent.com/81410437/174017890-c0abf9d0-6c1a-4595-be41-e14c6da3e428.mov

I could have made the change only for the "moveNodeBelow" function but for consistency's sake, I used the same method everywhere which is finding back the node's new row index after making the change (see changes in `EventsSheet/index.js` > `this._onDrop`.

Reply:
> I used the same method everywhere which is finding back the node's new row index after making the change

Very clear, and looks easier to understand. Thanks!